crypto/serializer/serializer_local.h

```diff
@@ -50,3 +72,58 @@ struct ossl_serializer_ctx_st {
      */
     UI_METHOD *allocated_ui_method;
 };
+
+struct ossl_deserializer_instance_st {
+    OSSL_DESERIALIZER *deser;    /* Never NULL */
+    void *deserctx;              /* Never NULL */
+    const char *input_type;      /* Never NULL */
+    const char *output_type;     /* Never NULL */
+};
+
+DEFINE_STACK_OF(OSSL_DESERIALIZER_INSTANCE)
+
+struct ossl_deserializer_ctx_st {
+    /*
+     * The caller may know the input type of the data they pass.  If not,
+     * this will remain NULL and the deserializing functionality will try
+     * whatever it has in store.
```

Comment:
what does 'it have in store' mean? Are you talking about OPENSSL_STORE??

Reply:
No, it's not meant literally like that. I'll rephrase.

Reply:
Done